sdk/identity/azure-identity/test/ut/azure_cli_credential_test.cpp

```diff
@@ -221,6 +221,56 @@ TEST(AzureCliCredential, ExpiresIn)
   EXPECT_LE(token.ExpiresOn, timestampAfter + std::chrono::seconds(30));
 }
 
+TEST(AzureCliCredential, ExpiresOnUnixTimestampInt)
```

Comment:
Can we add a test that results in an empty vector of strings being passed to `ParseToken` and also one where the vector of strings has elements that are `""`?